packages/framework/src/Component/EntityLog/EventListener/EntityLogEventListener.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\EntityLog\EventListener;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Event\PostFlushEventArgs;
+use Doctrine\ORM\Event\PostPersistEventArgs;
+use Doctrine\ORM\Event\PostUpdateEventArgs;
+use Doctrine\ORM\Event\PreRemoveEventArgs;
+use Exception;
+use Psr\Log\LoggerInterface;
+use Shopsys\FrameworkBundle\Component\EntityLog\Attribute\AttributeReader;
+use Shopsys\FrameworkBundle\Component\EntityLog\Attribute\LoggableSetup;
+use Shopsys\FrameworkBundle\Component\EntityLog\ChangeSet\ChangeSetResolver;
+use Shopsys\FrameworkBundle\Component\EntityLog\Enum\EntityLogActionEnum;
+use Shopsys\FrameworkBundle\Component\EntityLog\Model\EntityLogFacade;
+
+class EntityLogEventListener
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\EntityLog\Model\EntityLog[]
+     */
+    protected array $logs = [];
+
+    protected bool $enabled = true;
+
+    /**
+     * @param \Doctrine\ORM\EntityManagerInterface $em
+     * @param \Psr\Log\LoggerInterface $monolog
+     * @param \Shopsys\FrameworkBundle\Component\EntityLog\Attribute\AttributeReader $attributeReader
+     * @param \Shopsys\FrameworkBundle\Component\EntityLog\ChangeSet\ChangeSetResolver $changeSetResolver
+     * @param \Shopsys\FrameworkBundle\Component\EntityLog\Model\EntityLogFacade $entityLogFacade
+     */
+    public function __construct(
+        protected readonly EntityManagerInterface $em,
+        protected readonly LoggerInterface $monolog,
+        protected readonly AttributeReader $attributeReader,
+        protected readonly ChangeSetResolver $changeSetResolver,
+        protected readonly EntityLogFacade $entityLogFacade,
+    ) {
+    }
+
+    public function enable(): void
```

Comment:
It seems like it's no way of disabling it (the default value of the property is also `true`).
Maybe this method is not necessary?

Reply:
ill remove whole enabling ability.

Reply:
whole functionality was removed